src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -533,31 +538,24 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
      */
     private void adjustQuestionNumbers(int oldQuestionNumber,
             int newQuestionNumber, List<FeedbackQuestionAttributes> questions) {
-        
-        if (oldQuestionNumber > newQuestionNumber && oldQuestionNumber >= 1) {
-            for (int i = oldQuestionNumber - 1; i >= newQuestionNumber; i--) {
-                FeedbackQuestionAttributes question = questions.get(i - 1);
-                question.questionNumber += 1;
-                try {
+        try {
```

Comment:
Originally, there were 2 try blocks for each `updateFeedbackQuestionWithoutResponseRateUpdate(question);` , I abstracted those 2 out to this big block because they are trying the same thing anyway. Should I undo this change?